puppet/zulip_ops/files/nagios4/conf.d/hostgroups.cfg

```diff
@@ -53,7 +53,6 @@ define hostgroup {
 define hostgroup {
         hostgroup_name          redis
         alias                   Redis servers
-        hostgroup_members       frontends
```

Comment:
This commit message mentions the `nagios` hostgroup, but this is the redis hostgroup.

Reply:
Whoops!  Repushed.